src/env.h

```diff
@@ -263,6 +265,9 @@ class Environment {
     enum Fields {
       // Set this to not zero if the init hook should be called.
       kCallInitHook,
+      // Set this to not zero if async wrap should be enabled. This is
+      // automatically set when SetupHooks is called.
+      kEnabled,
```

Comment:
I think I have come to the conclusion that both behaviors are reasonable. However the `kCallInitHook` name is very confusion, as it suggests it only effects the init hook, while really it affects all hooks.